sycl/test/extensions/fpga.cpp

```diff
@@ -59,16 +59,14 @@ int main() {
   /*Check LSU interface*/
   {
     cl::sycl::buffer<int, 1> output_buffer(1);
-    cl::sycl::buffer<int, 1> input_buffer(1);
+    auto *in_ptr = cl::sycl::malloc_host<int>(1,Queue.get_context());
```

Comment:
> @aditikum, are you going to address the comments from @dm-vodopyanov and @mohammadfawaz?

I was planning to address both for a subsequent change pull request that would also include documentation update. Is it preferred to address both in this pull request?

Reply:
I think so. It's better to have self-contained changes rather than distributed among multiple patches.

- It looks like @dm-vodopyanov suggestions are just formatting fixes, so I think these better to be fixes within this PR.
- @mohammadfawaz, is it okay if your comment is addressed separately?

Reply:
I'm okay with a separate patch for the test changes.

Reply:
> > @aditikum, are you going to address the comments from @dm-vodopyanov and @mohammadfawaz?
> 
> I was planning to address both in a subsequent pull request that would also include documentation update. Is it preferred to address both in this pull request?

I addressed comments from @dm-vodopyanov. Please, apply suggestions from @mohammadfawaz in a separate pull request.